src/ess/amor/resolution.py

```diff
@@ -120,10 +120,11 @@ def angular_resolution(
 
 
```

Comment:
In the `sample_size_resolution`, we have
```Py
fwhm_to_std(
        sc.to_unit(sample_size, "m")
        / sc.to_unit(da.coords["position"].fields.z, "m", copy=False)
    )
```
but the paper says that the denominator should be the distance between sample and detector.

The current implementation only works is the sample is at the origin, so I guess we should have
```Py
fwhm_to_std(
        sc.to_unit(sample_size, "m")
        / sc.to_unit(sc.norm(da.coords["position"] - da.coords["sample_position"]), "m", copy=False)
    )
```
?

The same goes for the `angular_resolution`.